test/cf-log.test.ts

```diff
@@ -0,0 +1,1155 @@
+import * as cdk from '@aws-cdk/core';
```

Comment:
Also add/update test case in OpenTuna stack to make sure the cloudfront distribution specifying the access log destination